source/reducers/index.js

```diff
@@ -21,4 +22,12 @@ const rootReducer = combineReducers({
     remoteExplorer
 });
 
+const rootReducer = (state, action) => {
+    if (action.type === APP_MASTER_RESET) {
+        // reset global state
+        state = undefined;
```

Comment:
I think you have to return undefined instead of mutating the state here. Or is there a reason for this?

Reply:
Actually it's just modifying the local variable. Setting state to undefined here doesn't mutate state 😊

Reply:
I did take this example of a post online, hence the style.

Reply:
It's also nice because the `APP_MASTER_RESET` action can still be listened to, if need-be.

Reply:
Well if it works then probably its fine. I thought redux reducers need to return to set the state rather than setting value to it. But cool 😎👍